C/main.c

```diff
@@ -1,8 +1,100 @@
 #include <util/delay.h>
 #include "common/Types.h"
 #include "common/Utils.h"
+#include "hal/LCD/LCD_Interface.h"
 #include "hal/Buzzer/Buzzer.h"
+uint8_t heart[] =
+{
+  0x00,
+  0x00,
+  0x0A,
+  0x1F,
+  0x1F,
+  0x0E,
+  0x04,
+  0x00
+};
+uint8_t skull[8] =
+{
+0b00000,
+0b01110,
+0b10101,
+0b11011,
+0b01110,
+0b01110,
+0b00000,
+0b00000
+};
```

Comment:
we can add these value as a macro to header file to allow any one use them easily <br> e.g. 
```
#define HEART     0x00,\
                  0x00,\
                  0x0A,\
                  0x1F,\
                  0x1F,\
                  0x0E,\
                  0x04,\
                  0x00
```